anvil-python/anvil/commands/haproxy.py

```diff
@@ -100,34 +142,55 @@ def has_prompts(self) -> bool:
         return True
 
     def prompt(self, console: Console | None = None) -> None:
-        variables = questions.load_answers(
-            self.client, self._KEEPALIVED_CONFIG
-        )
+        variables = questions.load_answers(self.client, self._HAPROXY_CONFIG)
         variables.setdefault("virtual_ip", "")
+        variables.setdefault("charm_haproxy_config", {})
 
         # Set defaults
         self.preseed.setdefault("virtual_ip", "")
+        self.preseed.setdefault("charm_haproxy_config", {})
```

Comment:
`charm_haproxy_config` is a Terraform variable that we are keeping empty to give our users the options to always override what we set with our variables. It is provided as the last resort to skip all the sane defaults and whatever logic we have with single variables. e.g. `virtual_ip`. So I would like you to introduce dedicated variables for questions `ssl_cert`, `ssl_key`, similar Terraform variables for them and in the end, produce the services yaml with Terraform.

So here we will have something like:

```suggestion
        self.preseed.setdefault("ssl_cert", "")
        self.preseed.setdefault("ssl_key", "")
```

Reply:
moved to a different terraform variable, which merges with `charm_haproxy_config` (takes precedent)